roles/pulp/tasks/main.yaml

```
@@ -138,8 +141,21 @@
     command: pulpcore-manager migrate --noinput
     network: host
     secrets:
-      - 'pulp-settings-py,type=mount,target=/etc/pulp/settings.py'
       - 'pulp-symmetric-key,type=mount,target=/etc/pulp/certs/database_fields.symmetric.key'
+      - 'pulp-db-password,type=env,target=PULP_DATABASES__default__PASSWORD'
+    env:
+      PULP_ANALYTICS: "{{ pulp_enable_analytics | ternary('true', 'false') }}"
```

Until the Ansible issue is fixed, can we deduplicate where they are the same values by using `pulp_settings_env` as a dict?
```suggestion
      PULP_ANALYTICS: "{{ pulp_settings_env['PULP_ANALYTICS'] }}"
```

---

I've reduced the options to the ones needed for migrations